pkg/chunked/storage_linux.go

```diff
@@ -58,6 +58,20 @@ const (
 	copyGoRoutines = 32
 )
 
+// fileMetadata is a wrapper around internal.FileMetadata with additional private fields that
+// are not part of the TOC document.
+type fileMetadata struct {
+	internal.FileMetadata
+
+	// Chunks are computed by mergeTOCEntries.
```

Comment:
```suggestion
	// If FileMetadata.Type == TypeReg, this stores the TypeChunk entries relevant to this entry.
```
(“relevant to” can probably be improved upon)